### PR TITLE
Season statistics

### PR DESCRIPTION
**Pull request by elizakeating**
## Description

Added winningest_coach, worst_coach, most_accurate_team, least_accurate_team, most_tackles, and fewest_tackles methods to the SeasonStatistics class.
Refactored code and added helper methods.
Fixed .gitignore file.
Add require CSV to spec helper.

## Type of change

- [x] fix
- [x] feat
- [x] test
- [x] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] thorough testing has been implemented if adding feature
- [x] all tests pass

### Thanks!

### CODE REVIEW

File: lib/season_statistics.rb [hunk 1]
```diff
@@ -1,3 +1,178 @@
...
+    @locations = locations
+    @game_data = CSV.open locations[:games], headers: true, header_converters: :symbol
+    @teams_data = CSV.open locations[:teams], headers: true, header_converters: :symbol
+    @game_team_data = CSV.open locations[:game_teams], headers: true, header_converters: :symbol
+  end
+
+  def winningest_coach(season_id)
+    season_games = find_season_games(season_id)
+    season_game_teams = find_season_game_teams(season_games)
+
```
Comment (kbug819) [inline lib/season_statistics.rb:17]: This method is the WINNINGEST

**Comment by kbug819** [comment]
Great, the code looks clean. Great progress!

**Review by ethanb1145 — COMMENTED** [review]
I like the logic on the worst_coach and winningest coach methods, it's easy to follow and readable. All code looks neat and organized.

**Review by nurudeensulaimon — COMMENTED** [review]
Thanks for the clarity in your codes. We are winning bigly!

**Pull request merged**